meeting_minutes/2025/2025-03-26.md

```diff
@@ -0,0 +1,132 @@
+  - Omar sets a motion to accept the contribution of the quorom tool to use for calculating quorum and voting rights.
+  - Justin and Denny seconded. No discussion, no objections, motion passed.
+
+## GitHub Issues & Pull Requests
+
+- PR [#902](https://github.com/oasis-tcs/csaf/pull/902)
+  - Thomas has a few questions remaining re: CVSS v4.0 that he hopes to clarify at VulnCon with the FIRST CVSS SIG chair.
+    - Omar agrees to facilitate an introduction.
+  - Thomas discusses the conversion rule for CVSS v4.0 from CSAF v2.0 to CSAF v2.1.
+  - Thomas sets the motion to accept the conversion rule from CVSS v4.0 as stated in Pull Request [#902](https://github.com/oasis-tcs/csaf/pull/902).
+  - Justin seconded. No objections. No discussion, no objections, motion passed.
+- Issue [#884](https://github.com/oasis-tcs/csaf/issues/884)
+  - Thomas explains the need to include a `license` field in not only the `provider-metadata.json`, but also in the CSAF file itself.
+  - Sonny asks if the declaration of the license would be mandatory or an optional field.
+  - Thomas suggests making it a "SHOULD" statement.
+  - Omar says he does not like the suggestion of including `license` in a CSAF file, and prefers licensing to be referenced externally.
+  - Thomas clarifies this is not about the license of the product, but a license of the data that is in the CSAF document.
+  - Christian agrees with including licensing in individual documents and not just in the `provider-metadata.json`, since it is not simple to navigate from a CSAF file to the `provider-metadata.json`. Could be difficult to find the license if it is not specified in the file.
+  - Christian asks whether the existing legal disclaimer would suffice, although it might be easier having a dedicated field, suchs as an SPDX identifier.
+  - Sonny asks would it be possible to split up the `legal_disclaimer` into multiple `legal_disclaimer` fields of certain types as an implementation option?
+  - Thomas notes that multiple `legal_disclaimer` fields can already be used, such as one for "terms of use" or a "classic" legal disclaimer.
+  - Thomas mentions the value of having the dedicated `license` field is for the purpose of machine-readability. Human readable licenses do not scale with automation.
+  - Thomas references the BSI technical guidelines for using license identifiers, including guidance for cases when no appropriate identifier exists.
+  - Sonny asks for clarification where the `license` field would be in the document.
+  - Thomas specifies under "Distribution" have a new property called `license`, and would hold the identifier.
+  - Sonny asks if will it be mandatory, prompting Thomas to ask the TC who would prefer that this information should be mandatory?
+  - Omar highly suggests it should not be mandatory.
+  - Thomas moves to add a `license` identifier under "Distribution" for CSAF v2.1. Omar modifies the motion to specify that this is an optional field.
```

Comment:
@justmurphy Could you please double check whether the comment below is true? Just trying to clarify the motion...
```suggestion
  - Thomas moves to add a `license` identifier under "Distribution" for CSAF v2.1. 
  - Omar suggests to modify the motion to specify that this is an optional field.
  - Thomas  agrees and moves to add a `license` identifier under "Distribution" for CSAF v2.1 as an optional field.